File: androidProject/POSRentalPS/app/src/main/java/com/theputras/posrentalps/model/PaketSewa.java
package com.theputras.posrentalps.model;
import com.google.gson.annotations.SerializedName;

public class PaketSewa {
    @SerializedName("id_paket")
    public int idPaket;
    @SerializedName("nama_paket")
    public String namaPaket;
    @SerializedName("durasi_menit")
    public int durasiMenit;
    @SerializedName("harga")
    public int harga;
    @SerializedName("nama_console")
    public String namaConsole; // Dari join table di backend
}

File: laravelProject/app/Models/PaketSewa.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class PaketSewa extends Model
{
    use HasFactory;

    // Karena nama tabelnya bukan 'paket_sewas' (default laravel), kita harus define
    protected $table = 'paket_sewa';
    
    // Karena primary key-nya 'id_paket' (bukan 'id'), kita harus define
    protected $primaryKey = 'id_paket';

    protected $fillable = [
        'id_console',
        'nama_paket',
        'durasi_menit',
        'harga'
    ];

    // Relasi ke Jenis Console (Opsional, tapi bagus buat validasi)
    public function console()
    {
        return $this->belongsTo(JenisConsole::class, 'id_console', 'id_console');
    }
}